introduction.md

```diff
@@ -2,3 +2,22 @@
 Hi, I'm Clive. I'm a Web Developer/contractor in the UK. I work on allsorts of projects.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?